resources/features/Sampling.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Making a Sample
+    The sample creation method will ingest a frame and a set of stratification parameters to produce an output sample.
+    To do this, the pre-filtered dataset is sorted in ascending order by 'prn'. Each of the stratification stratas with
+    a selection type parameter set as 'Census' or 'Prn Sampling' (ommitting 'Universal') is then processed. We then use
+    the start point parameter (of a given strata), and select units in prn order until the required sample size is
```

Comment:
There is no requirement to use PRN points for a 'Census' cell as all units are selected. The stratification parameters hold a dummy PRN value of 0.000000000 and number required value of 0 for the 'Census' cells to reflect this.
For clarity we should state that where the selection type parameter is set to 'Census', all units are selected; where the type is 'PRN Sampling' we use the start point and select units in PRN order until the number required is reached.

Reply:
Good point - seems I didn't mention a critical part